db/migrate/20230919195907_create_post.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :author, null: false, foreign_key: { to_table: :users }
+      t.string :title
+      t.string :text
```

Comment:
- [ ] It would be nice if you could make the type of the post `text` field be of type `text` as it will be taking a lengthy text about a post and the `text` type is best for that instead of the `string` type. Kindly use `t.text :text` 👍🏽